common/sgx/quote.c

```diff
@@ -27,12 +27,17 @@
 #include "../../host/sgx/quote.h"
 #endif
 
+#ifdef OEUTIL_TCB_ALLOW_ANY_ROOT_KEY // allow overrode by oeutil
+// Defined by tools/oeutil/host/generate_evidence.cpp
```

Comment:
It seems as if a host file (generate_evidence.cpp) is getting pulled into the TCB.
Will `OEUTIL_TCB_ALLOW_ANY_ROOT_KEY` be used only when building the host?

Reply:
yes, it's a special oehost that's only used by oeutil
https://github.com/openenclave/openenclave/blob/3ec066b43aaab67500be48ea77a16ee91c3050d3/tools/oeutil/host/CMakeLists.txt#L26